### PR TITLE
check properly content-type

### PR DESCRIPTION
**Pull request by AlvaroVega**
most of the times content-type is something like: `application/json; utf-8` which was detected as unknown content-type
As a result of then a log with a false failure was printed.

### CODE REVIEW

File: lib/fiware-pep-steelskin.js [hunk 1]
```diff
@@ -200,12 +200,16 @@ function traceRequest(req, res, next) {
...
-        logger.debug('Body:\n\n%s\n\n', JSON.stringify(req.body, null, 4));
-    } else if (req.headers['content-type'] === 'application/xml') {
-        logger.debug('Body:\n\n%s\n\n', req.rawBody);
+    if (req.headers['content-type']) {
+        if (req.headers['content-type'].includes('application/json')) {
+            logger.debug('Body:\n\n%s\n\n', JSON.stringify(req.body, null, 4));
+        } else if (req.headers['content-type'].includes('application/xml')) {
+            logger.debug('Body:\n\n%s\n\n', req.rawBody);
+        } else {
+            logger.debug('Unrecognized body type', req.headers['content-type']);
```
Comment (fgalan) [inline lib/fiware-pep-steelskin.js:209]: Some operations in the CB API use text/plain as content-type. In particular this one:

```
PUT /v2/entities/{E}/attrs/{A}/value
```

which corresponds to this action https://github.com/telefonicaid/fiware-pep-steelskin/blob/master/lib/plugins/orionUrls.js#L112

Could be problematic taking into account this logic?

Comment (fgalan) [reply lib/fiware-pep-steelskin.js:209]: I have checked that actually text/plain is being used in our reference postman collection:

https://github.com/telefonicaid/thinking-cities/blob/master/postman/ThinkingCities%20APIs%20Basic%20Ops.postman_collection.json#L770-L771

Comment (AlvaroVega) [reply lib/fiware-pep-steelskin.js:209]: Really there is no problem, this funcion is just to log request.

Comment (fgalan) [reply lib/fiware-pep-steelskin.js:210]: Fixed in 0e64b62

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -3,3 +3,4 @@
 - Upgrade requests dep from 2.88.0 to 2.88.2
 - Fix: propagate correlator in FWD request (#468)
 - Fix: propagate correlator in validation (Access Control) requests
+- Fix: check properly content-type of request
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:6]:
```suggestion
- Fix: log properly content-type of request
```

As it seems this fix is only related with the logs ssytem

**Review by fgalan — APPROVED** [review]
LGMT

**Pull request merged**